changes/v4.4.11-en.md

```diff
@@ -1,5 +1,8 @@
 ### Enhancements
 
+- OTP upgrade from 24.1.5-3 to 24.3.4.2-1
+[#9265](https://github.com/emqx/emqx/pull/9265)
```

Comment:
```suggestion
- OTP upgrade from 24.1.5-3 to 24.3.4.2-1 [#9265](https://github.com/emqx/emqx/pull/9265).
  Change highlights:
    - Erlang/OTP [SSL library vulnerability fix](https://nvd.nist.gov/vuln/detail/CVE-2022-37026)
    - Added support for OCSP (Online Certificate Status Protocol) Stapling
    - Added CRL (Certificate revocation list) cache auto refresh
```

Reply:
shall we state that the hot upgrade will not load the New OTP?